scripts/create_mailgun_route.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+"""Create mailing lists for all top-level projects
+"""
+
+import sys
+import logging
+import requests
+
+from website.app import init_app
+from website.util import api_url_for
+from website.settings import MAILGUN_API_KEY, SHORT_DOMAIN, DOMAIN
+
+logger = logging.getLogger(__name__)
+
+
+# Warning: This script is not in a ready state.
+def main():
+    init_app()
+    dry = 'dry' in sys.argv
+    if not dry:
+        requests.post(
+            "https://api.mailgun.net/v3/routes",
+            auth=("api", MAILGUN_API_KEY),
+            data={"priority": 0,
+                  "description": "Project Mailing Route",
+                  "expression": "match_recipient('.*@{}')".format(SHORT_DOMAIN),
```

Comment:
The Domain here needs to be changed to one that will work on all OSF versions in the form \<guid\>@\<domain\>.